<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-K42ITB2" failures="1" tests="1" name="Default test(failed)(failed)" time="19.564" errors="0" timestamp="2023-03-02T10:31:02 IST">
  <testcase classname="Day5.TestScript05" name="testCase05" time="2.643">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at Day5.TestScript05.testCase05(TestScript05.java:116)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testCase05 -->
</testsuite> <!-- Default test(failed)(failed) -->
